### PR TITLE
clash-1.1.0; v2ray-desktop-2.1.7

### PR DESCRIPTION
**Pull request by Nexus01**
clash-build-fix
v2ray-desktop-build-fix

### CODE REVIEW

**Comment by MingcongBai** [comment]
This pull request will have to be reopened when this cycle ends (topic-based).

**Comment by Nexus01** [comment]
> This pull request will have to be reopened when this cycle ends (topic-based).

OK

File: extra-network/clash/autobuild/defines [hunk 1]
```diff
@@ -3,4 +3,3 @@ PKGSEC=net
 PKGDEP="glibc"
 BUILDDEP="go"
 PKGDES="A rule-based tunnel for VMess, Shadowsocks, Trojan, Snell protocols"
-ABTYPE=gomod
```
Comment (MingcongBai) [inline extra-network/clash/autobuild/defines:6]: Did this somehow not work with your first PR that introduced this package?

File: extra-network/v2ray-desktop/autobuild/prepare [hunk 2]
```diff
@@ -1 +1,4 @@
 acbs_copy_git
+abinfo "Set the v2ray_usr_local_install..."
```
Comment (MingcongBai) [inline extra-network/v2ray-desktop/autobuild/prepare:2]: This is not descriptive enough of the operations applied below.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-network/clash/autobuild/defines (hunk 1 above)
Comment (Nexus01) [reply extra-network/clash/autobuild/defines:6]: Because `gomod` cannot pass the version message to `clash`. Then, I use the `build` file.

**Comment by MingcongBai** [comment]
Closing - this is *not* a rejection.

We are switching to a topic-based maintenance system. Please re-create your PR against our `stable` branch, use our new Pull Request template, and work in accordance to our [maintenance guidelines](https://wiki.aosc.io/developer/packaging/topic-based-maintenance-guideline/).

**Pull request closed without merging**